server/src/server.js

```diff
@@ -122,6 +122,7 @@ if (config.enableCoverage && istanbulMiddleware) {
 }
 
 const CONTENT_NAME = config.contentOrigin || '';
+const FXA_SERVER = config.fxa.server;
```

Comment:
Now that I realize that this is the same as the profileServer, I think this should just be `config.fxa.profileServer && require("url").parse(config.fxa.profileServer).host`